### PR TITLE
Update to SDK 5.2.2 - Closes #1464

### LINKED ISSUE #1464: Update SDK legacy docs for 5.2.2
### Which content is outdated?
- Version 5.2.1 
- Node.JS version 12
- Lisk Commander 5.1.7
- lisk-client 5.2.1

### What should be the new content?
- Version 5.2.2
- Node.JS version 16
- Lisk Commander 5.1.10
- lisk-client 5.2.2

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1464

### CODE REVIEW

File: modules/ROOT/pages/guides/app-development/setup.adoc [hunk 1]
```diff
@@ -97,7 +97,7 @@ If you are using NVM (Node.js Version Manager), install the correct version as s
 
 [source,bash]
 ----
-nvm install v12.22.9
+nvm install v16.15.0
 ----
 
 NOTE: The correct version for NPM(6) is installed automatically with version 12 of Node.js.
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/setup.adoc]:
```suggestion
NOTE: The latest version for NPM(6) is installed automatically with version 12 of Node.js.
```

File: modules/ROOT/pages/guides/app-development/setup.adoc [hunk 2]
```diff
@@ -410,13 +410,13 @@ How to create a new blockchain application manually without using the Lisk Comma
 
 === Dependencies
 
-* Node.js v12.22.9 (NPM version 6.14.15)
+* Node.js v16.15.0
 
 If you are using NVM, install the correct version as shown below:
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/setup.adoc]:
```suggestion
If you are using NVM, install the latest version as shown below:
```

File: modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -100,7 +100,7 @@ If you are using NVM, (Node.js Version Manager), ensure you install the correct
 
 [source,bash]
 ----
-nvm install v12.22.9
+nvm install v16.15.0
 ----
 
 NOTE: The correct version for NPM(6) is installed automatically with version 12 of Node.js.
```
Comment (chris529) [inline modules/ROOT/pages/index.adoc]:
```suggestion
NOTE: The latest version for NPM(6) is installed automatically with version 12 of Node.js.
```

**Review by chris529 — COMMENTED** [review]
Looks good, just the minor points we discussed 👍

File: antora.yml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 name: lisk-sdk
 title: Lisk SDK
 version: 'master'
```
Comment (ishantiw) [inline antora.yml:3]:
```suggestion
version: 'main'
```

File: modules/ROOT/pages/references/lisk-commander/commands.adoc [hunk 5]
```diff
@@ -21,7 +21,7 @@ Displays command reference.
 A command line interface for Lisk
 
 VERSION
-  lisk-commander/5.1.7 darwin-x64 node-v12.20.1
+  lisk-commander/5.1.10 darwin-x64 node-v16.5.0
```
Comment (ishantiw) [inline modules/ROOT/pages/references/lisk-commander/commands.adoc]:
```suggestion
  lisk-commander/5.1.10 darwin-x64 node-v16.15.0
```

**Review by ishantiw — CHANGES_REQUESTED** [review]
LGTM, couple of minor changes

File: antora.yml (hunk 4 above)
Comment (Tschakki) [reply antora.yml:3]: This should stay, `master` is referring to antora-internal versioning. For Antora v3, it could now be replaced with ~, or 5.2.2

**Review by ishantiw — APPROVED** [review]

**Pull request merged**